cilium/cmd/bpf_proxy_list.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cilium/cilium/common"
+	"github.com/cilium/cilium/pkg/bpf"
+	"github.com/cilium/cilium/pkg/proxy"
+)
+
+var proxyList = map[string]string{}
+
+// bpfProxyListCmd represents the bpf_proxy_list command
+var bpfProxyListCmd = &cobra.Command{
+	Use:   "list",
+	Short: "List proxy configuration",
+	Run: func(cmd *cobra.Command, args []string) {
+		common.RequireRootPrivilege("cilium bpf proxy list")
+		proxy.Dump(dumpProxy4)
+		proxy.Dump6(dumpProxy6)
+
+		if len(dumpOutput) > 0 {
+			if err := OutputPrinter(proxyList); err != nil {
+				os.Exit(1)
+			}
+			return
+		}
+
+		title := fmt.Sprintf("%-20s %-s", "Proxy", "Destination")
```

Comment:
Some of the other CLI tools use tabwriter, and I think it's a good way to ensure that the columns being outputted line up. For example, with IPv6 it's easy for addresses to exceed 20 characters.

Would you mind updating this to do the same? Here's an example:

https://github.com/cilium/cilium/blob/b2f922369413d43ac592706f444af5dabf085ce9/cilium/cmd/bpf_policy_list.go#L82
https://github.com/cilium/cilium/blob/b2f922369413d43ac592706f444af5dabf085ce9/cilium/cmd/bpf_policy_list.go#L115
https://github.com/cilium/cilium/blob/b2f922369413d43ac592706f444af5dabf085ce9/cilium/cmd/bpf_policy_list.go#L134